package.json

```diff
@@ -197,6 +199,7 @@
     "regenerator-runtime": "^0.10.5",
     "selenium-webdriver": "^3.6.0",
     "sqlite3": "^4.1.1",
+    "supertest": "^6.2.3",
```

Comment:
What gains are you getting from using `supertest` vs the current libraries present in the codebase?

Reply:
The main feature I am leveraging from supertest is the ability to test out http requests in process without having to set up a server. I am using this feature in a test to allow me to set up the configuration params of the JWT issuer to be something unique to the test, and supertest allows me to assert on the response.

Basically these two assertions here:
```js
       // Login callback will send user to the appropriate page with the right token
        await request(app)
          .get(`/login-callback?nextUrl=/admin/${orgId}/campaigns`)
          .set("Authorization", `Bearer ${signedToken}`)
          .expect(302)
          .expect("Location", `/admin/${orgId}/campaigns`);

        // Bad tokens get rejected
        await request(app)
          .get(`/login-callback`)
          .set("Authorization", `Bearer ${invalidToken}`)
          .expect(401);
 ```